test/test.user.js

```diff
@@ -54,6 +54,13 @@ describe('Github.User', function() {
       });
    });
 
+   it('should get user.orgMembers from "dolores" company', function(done) {
```

Comment:
I'm not sure what this dolores company is but probably we should use another one. A better fit would be the [GitHub Tools organization](https://github.com/github-tools).